### PR TITLE
Fix login form validation

### PR DESCRIPTION
**Pull request by JJthere**
## Description

This PR fixes the login form validation.

### CODE REVIEW

File: frontend/components/form/InputField.tsx [hunk 1]
```diff
@@ -33,6 +33,10 @@ interface InputFieldProps {
 	 * Is the InputField required?
 	 */
 	required?: boolean;
+	/**
+	 * Minimal input length
+	 */
+	minLength?: number;
```
Comment (darmiel) [inline frontend/components/form/InputField.tsx:39]: do you use this prop? It appears that this PR doesn't make use of this.

Comment (JJthere) [reply frontend/components/form/InputField.tsx:39]: I do not use it. It will be used in later updates though.

Comment (darmiel) [reply frontend/components/form/InputField.tsx:39]: alright

**Review by darmiel — APPROVED** [review]

**Review by memeToasty — APPROVED** [review]

**Review by Julian702 — APPROVED** [review]
Well done!

**Review by 4KevR — APPROVED** [review]

**Review by TomRomeo — APPROVED** [review]

**Pull request merged**